portal/app/farm_updater/egauge_api.py

```diff
@@ -69,6 +69,7 @@ def request_egauge_data(self):
     def processing_egauge_data(self):
         from app.models import FarmDevice
         from app.models import DeviceType
+        from app.models import PowernetUserType
```

Comment:
Where are you using PowernetUserType? Seems like we can get rid of it

Reply:
forgot to remove it